doc/developers/develop.rst

```diff
@@ -217,20 +217,6 @@ Optional Arguments
 In iterative algorithms, the number of iterations should be specified by
 an integer called ``n_iter``.
 
-Pairwise Attributes
-^^^^^^^^^^^^^^^^^^^
-
-An estimator that accepts ``X`` of shape ``(n_samples, n_samples)`` and defines
-a :term:`_pairwise` property equal to ``True`` allows for cross-validation of
-the dataset, e.g. when ``X`` is a precomputed kernel matrix. Specifically,
-the :term:`_pairwise` property is used by ``utils.metaestimators._safe_split``
-to slice rows and columns.
```

Comment:
I think we can keep the information about `safe_split` and move it into https://github.com/scikit-learn/scikit-learn/blob/d017f6fdc66c9000e2c867bf9166f3163fc8755c/doc/developers/develop.rst?plain=1#L547